test/end-to-end/transactionType/transactionType.spec.js

```diff
@@ -0,0 +1,119 @@
+const components = require('../shared/components');
+
+helpers.configure(chai);
+
+describe('Transaction Types ::', () => {
+
+  // navigate to the page
+  before(() => helpers.navigate('#/admin/transaction_type'));
+
+  // after the creation the inserted id will be 11
+  let identifier = 11;
+
+  let newType = {
+    text: 'E2E Transaction Type',
+    description: 'Test transaction type',
+    type: 'income',
+    prefix: 'E2ETT'
+  };
+
+  let updateType = {
+    text: 'E2E Transaction Type updated',
+    description: 'Test transaction type updated',
+    type: 'expense',
+    prefix: 'E2ETT_UPDATED'
+  };
+
+  let otherType = {
+    text: 'E2E Other Transaction Type',
+    description: 'Test Other transaction type',
+    type: 'other',
+    prefix: 'E2EOTT',
+    other: 'OTHER_TYPE'
+  };
+
+  it('Successfully creates a transaction type', () => {
+    FU.buttons.create();
+    FU.input('$ctrl.transactionType.text', newType.text);
+    FU.input('$ctrl.transactionType.description', newType.description);
+    FU.input('$ctrl.transactionType.prefix', newType.prefix);
+    element(by.model('$ctrl.transactionType.type')).click();
+    element(by.css(`[value=${newType.type}]`)).click();
+    FU.buttons.submit();
+    components.notification.hasSuccess();
+  });
+
+  it('Successfully updates an existing transaction type', () => {
+    element(by.css('[data-edit-type="' + identifier + '"]')).click();
+    FU.input('$ctrl.transactionType.text', updateType.text);
+    FU.input('$ctrl.transactionType.description', updateType.description);
+    FU.input('$ctrl.transactionType.prefix', updateType.prefix);
+    element(by.model('$ctrl.transactionType.type')).click();
+    element(by.css(`[value=${updateType.type}]`)).click();
+    FU.buttons.submit();
+    components.notification.hasSuccess();
+  });
+
+  it('Successfully creates a transaction type with a specific type', () => {
+    FU.buttons.create();
+    FU.input('$ctrl.transactionType.text', otherType.text);
+    FU.input('$ctrl.transactionType.description', otherType.description);
+    FU.input('$ctrl.transactionType.prefix', otherType.prefix);
+    element(by.model('$ctrl.transactionType.type')).click();
+    element(by.css(`[value=${otherType.type}]`)).click();
+    FU.input('$ctrl.otherType', otherType.other);
+    FU.buttons.submit();
+    components.notification.hasSuccess();
+  });
+
+  it('Dont creates a new transaction type for missing type', () => {
+    FU.buttons.create();
+    element(by.model('$ctrl.transactionType.type')).click();
+    element(by.css(`[value=${otherType.type}]`)).click();
+    FU.buttons.submit();
+
+    // check validations
+    FU.validation.error('$ctrl.transactionType.text');
+    FU.validation.error('$ctrl.transactionType.description');
+    FU.validation.error('$ctrl.transactionType.prefix');
+    FU.validation.error('$ctrl.otherType');
+
+    // be sure not success
+    expect(element(by.css('[data-notification-type="notification-success"]')).isPresent())
```

Comment:
You can remove this line.  If it has a danger notification (line 96 asserts it), it will not have a success notification.